src/locale/locales/fr/messages.po

```diff
@@ -634,11 +596,11 @@ msgstr "Modifier votre e-mail"
 #: src/screens/Deactivated.tsx:72
 #: src/screens/Deactivated.tsx:76
 msgid "Check my status"
-msgstr ""
+msgstr "Vérifier mon statut"
 
 #: src/view/com/auth/onboarding/RecommendedFeeds.tsx:121
 msgid "Check out some recommended feeds. Tap + to add them to your list of pinned feeds."
-msgstr "Consultez quelques fils d’actu recommandés. Appuyez sur + pour les ajouter à votre liste de fils d’actualité."
+msgstr "Consultez quelques fils d’actualité recommandés. Appuyez sur + pour les ajouter à votre liste de fils d’actu."
```

Comment:
We settled on using `fils d’actu` everywhere for `feeds`, so I don’t understand the value of expanding `actu` into `actualité` here. I don’t think any French speaking user would be puzzled by the meaning of `actu` in its shorten form (it’s common, and even have it’s own entry in [Le Larousse](https://www.larousse.fr/dictionnaires/francais/actu/10910124) or [the French-speaking Wiktionnary](https://fr.wiktionary.org/wiki/actu)).

(still, good catch that the second use here wasn't shortened! Glad it's fixed thanks to this PR :))

```suggestion
msgstr "Consultez quelques fils d’actu recommandés. Appuyez sur + pour les ajouter à votre liste de fils d’actu."
```